sccp/figures/figureS1.py

```diff
@@ -0,0 +1,77 @@
+"""
```

Comment:
Remove this from the PR. You can keep it as its useful code, but this file does not create a figure. Alternatively, this could be output within a file that creates one of the other figures, if you find it would be very useful to keep around.